Example Code/Reflowduino_Demo/Reflowduino_Demo.ino

```diff
@@ -64,17 +66,26 @@ Adafruit_MAX31855 thermocouple(MAX_CS);
 
 // Define reflow temperature profile parameters (in *C)
 // First define a subtraction constant to compensate for overshoot:
-#define T_const 5 // From testing, overshoot was about 5-6*C
+#define T_const 1 // From testing, overshoot was about 5-6*C
 
 // Standard lead-free solder paste (melting point around 215*C)
 //#define T_preheat 150
 //#define T_soak 217
 //#define T_reflow 249 - T_const
 
 // "Low-temp" lead-free solder paste (melting point around 138*C)
-#define T_preheat 90
-#define T_soak 138
-#define T_reflow 165 - T_const
+//#define T_preheat 90
+//#define T_soak 138
+//#define T_reflow 165 - T_const
+
+// Superior Solder 8013-85 Water Soluble
+#define T_preheat 120
+#define T_soak 150
+#define soak_time 50*1000
```

Comment:
I chose not to do time because most users probably don't know what time to use, and the reflow curves always follow rate of temperature increase/decrease and temperature set points, not time. This way you have more control over the temperature profile to eliminate the risk of thermal shock on components. With time it's much harder to gauge how quickly it's really going to be. Also, if the oven simply isn't capable of heating up quickly enough, it won't be able to run within the time constraints. However, by simply defining a max temp increase/decrease rate the user can be assured that it won't hurt components and the toaster oven can stay below that value if it's not capable of reaching it.

Reply:
yeah, they definitely follow rate, the datasheet made me think that letting the components stabilize at the temperature was a good idea. It might only be useful in the soak, not in the reflow stages though.